meeting/adapter/command/command.go

```diff
@@ -10,9 +10,10 @@ import (
 	"github.com/WeisswurstSystems/WWM-BB/wwm"
 	"net/http"
 	"github.com/WeisswurstSystems/WWM-BB/meeting/usecase/toggleorderpayed"
+	"github.com/WeisswurstSystems/WWM-BB/meeting/usecase/invite"
 	"github.com/gorilla/mux"
-	"github.com/go-errors/errors"
 	"github.com/WeisswurstSystems/WWM-BB/meeting"
+	"github.com/go-errors/errors"
```

Comment:
Das hattest du schön öfter drin. Willst du absichtlich die stacktrace fehler haben?

Ich eigentlich nicht. Das normale Paket heißt "errors"

Reply:
das habe ich reingenommen, weil es nach deiner Umstrukturierung in einem File drin war. Kann ich gerne rausnehmen!